src/js/components/Box/ResponsiveContainer.js

```diff
@@ -0,0 +1,63 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import {
+  ResponsiveContainerContext,
+  ResponsiveContext,
+  ThemeContext,
+} from '../../contexts';
+import { StyledResponsiveContainer } from './StyledResponsiveContainer';
+import { deviceResponsive, getBreakpoint } from '../../utils';
+
+const responsiveContainerValue = true;
+
+export const ResponsiveContainer = ({ children }) => {
+  const theme = useContext(ThemeContext);
+  const ref = useRef(undefined);
+  const [stateResponsive, setResponsive] = useState();
```

Comment:
```suggestion
  const [stateResponsive, setStateResponsive] = useState();
```

Reply:
if we change this then have to change below as well

Reply:
Done